solutions/ys-SortPointsByArgument.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: ys-SortPointsByArgument
+source: YS
+title: Sort Points by Argument
+author: Jeffrey Zhang
+---
+
+## Solution
+
+The idea for this problem is fairly straightforward. Given $n$ points $\{P_1,...,P_n\}$, we can find the angle of the point $P_i = (x_i,y_i)$ by calling $\tan^{-1}{\frac{y_i}{x_i}}$. To sort the points in the counterclockwise order, we just sort the points based on their angle from smallest to largest. 
```

Comment:
"based on their angle"
elaborate?

Reply:
"we can find the angle of the point $P_i = (x_i,y_i)$ by calling $\tan^{-1}{\frac{y_i}{x_i}}$."
The angle of a point is referring to this sentence.

Reply:
is that measurement actually called the angle of a point?

Reply:
Wait my bad, meant to say "angle of a point from the origin", now this is an actual thing

we can find the angle of the point $P_i = (x_i,y_i)$ from the origin by calling $\tan^{-1}{\frac{y_i}{x_i}}$. To sort the points in the counterclockwise order, we just sort the points based on their angles from the origin from smallest to largest. 

Is this better?

Reply:
i'll wait for a second opinion actually